packages/venia-ui/lib/components/QuantityStepper/QuantityStepper.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from 'react';
```

Comment:
Wouldn't be an inaugural code review without some trivial naming convention feedback; for consistency, suggest we switch filenames to camelCase, eg. `quantityStepper.js`.